lib/tasks/plantain.rake

```diff
@@ -19,13 +19,13 @@ namespace :plantain do
   end
 
   desc "Run Solr and Arclight for interactive development"
-  task :development, %i[rails_server_args] do |_t, args|
+  task :development do
```

Comment:
I'm proposing this given that I kept encountering problems when trying to run the entire Solr/Rails stack with a single Rake Task in my environment (e. g. there would be false port collisions for the `solr_wrapper` Solr installation).